index.html

```diff
@@ -65,12 +65,15 @@
 ## Modal verbs terminology
 In the present document "shall", "shall not", "should", "should not", "may", "need not", "will", "will not", "can" and "cannot" are to be interpreted as described in clause 3.2 of the ETSI Drafting Rules (Verbal forms for the expression of provisions).
 
+## Definition of mnemonics and syntax function
+In the present documents the mnemonics, the syntax functions, and the syntax descriptors are to be interpreted as described in ([[MPEG-2 TS]]). The **uimsbf** and **bslbf** mnemonics are defined in Section 2.2.6 of ([[MPEG-2 TS]]). The **nextbits()** function is interpreted as in ([[MPEG-2 TS]]).
+
 <section class="normative">
  <h1>Identifying AV1 streams in MPEG-2 TS</h1>
 
-## AV1 Registration Descriptor
+## AV1 registration descriptor
 
-The presence of a Registration Descriptor, as defined in [[!MPEG-2 TS]], is mandatory with the *format_identifier* field set to 'AV01' (A-V-0-1). The Registration Descriptor shall be the first in the PMT loop and embeds the AV1 video subdescriptor.
+The presence of a Registration Descriptor, as defined in [[!MPEG-2 TS]], is mandatory with the *format_identifier* field set to 'AV01' (A-V-0-1). The Registration Descriptor shall be the first in the PMT loop and included before an AV1 video descriptor.
```

Comment:
shouldn't it be "before the AV1 descriptor"?

Reply:
Fix in f34d1770af58b39e976942d8b1f2152d943c5a76